### ISSUE TITLE
Add term 3-4 toe cutaneous syndactyly

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10406]
**Preferred term label:**
3-4 toe cutaneous syndactyly

**Synonyms**
Soft tissue syndactyly of toes 3 and 4 - Webbed 3rd and 4th toes - Webbed third and fourth toes

**Definition (free text, please give PubMed ID)**
[based on 2-4 toe cutaneous syndactyly HP:0005768]
A soft tissue continuity in the anteroposterior axis between the toes 3 and 4.

**Parent term (use hpo.jax.org/app)**
Cutaneous syndactyly of toes HP:0010621

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
For example: PMID25251565
Carpenter syndrome (omim201000)

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@lisannesw 
Adding new term: 
- ID: HP:6000648
- label: 3-4 toe cutaneous syndactyly
- parents: [Cutaneous syndactyly of toes (HP:0010621)](https://hpo.jax.org/app/browse/term/HP:0010621)
- pmid: PMID:25251565
- def: A soft tissue continuity in the anteroposterior axis between the toes 3 and 4.
- comment: 
- orcid: orcid:0009-0007-1020-5412
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10406
- synonyms:
